### PR TITLE
updates to support RHEL 8 playbooks

### PR DESCRIPTION
**Pull request by sej-jackson**
Added tasks for RHEL8 to install Common packages, and skip CPAN & gmake roles for issue #1125.

### CODE REVIEW

**Comment by sej-jackson** [comment]
I believe the `libdwarf-devel` and `libmpc-devel` packages need to come from another repository - possibly `ftp3-CRB`?

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@
 # All
 - name: Install JSON
   command: cpanm --with-recommends --sudo JSON
+  when:
+    - not ((ansible_distribution == "RedHat" or ansible_distribution == "CentOS") and ansible_distribution_major_version == "8")
```
Comment (gdams) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml]: @smlambert Am I correct in saying that we no longer need these cpan packages at all? If so we should just remove this role.

**Comment by sxa555** [comment]
We should ideally try and put these into `CentOS.yml` files in the `Common` role too to keep them in sync with RHEL

**Comment by sej-jackson** [comment]
> We should ideally try and put these into `CentOS.yml` files in the `Common` role too to keep them in sync with RHEL

Yes, @Willsparker is going to do them - he's just helping me with my changes at the moment....

**Comment by Willsparker** [comment]
I've been having issues with CentOS8, namely this: 
```
# sudo yum upgrade
CentOS-8 - Base                                                                                           1.2 kB/s | 2.9 kB     00:02    
Error: Failed to synchronize cache for repo 'BaseOS'
```
Not sure what the issue is, but I'll put the CentOS8 changes into a separate PR

**Comment by Willsparker** [comment]
If nobody objects, I'm going to open a separate PR for CentOS8 changes , as I'll take the opportunity to add a CentOS8 Vagrantfile at the same time :-)

**Comment by sej-jackson** [comment]
> If nobody objects, I'm going to open a separate PR for CentOS8 changes , as I'll take the opportunity to add a CentOS8 Vagrantfile at the same time :-)

Go for it. :)

**Comment by Willsparker** [comment]
Squashed and merged - the non-Common role changes will be tested regularly with the CentOS8 PR is merged with the `VagrantPlaybookCheck` Jenkins job. The `Common` role changes were tested before on an internal machine too :)

**Pull request merged**